src/components/structures/auth/Login.js

```diff
@@ -573,9 +575,12 @@ module.exports = createReactClass({
         // If this bug gets fixed, it will break SSO since it will open the SSO page in the
         // user's browser, let them log into their SSO provider, then redirect their browser
         // to vector://vector which, of course, will not work.
-        return (
+        return <div>
```

Comment:
please keep the parens - it helps with readability.

Reply:
I wonder if we should retro on parens around JSX. (I have so far been removing them whenever I have encountered them...)

Reply:
we did at some point and said we should be adding them 😭

Reply:
Ah, I see... We should record said fact on the code style then, I guess.